packages/firebase_crashlytics/firebase_crashlytics/pubspec.yaml

```diff
@@ -2,26 +2,26 @@ name: firebase_crashlytics
 description:
   Flutter plugin for Firebase Crashlytics. It reports uncaught errors to the
   Firebase console.
-version: 0.4.0+1
+version: 0.5.0-nullsafety.0
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_crashlytics
 
 environment:
   sdk: ">=2.12.0-0 <3.0.0"
   flutter: ">=1.12.13+hotfix.5 <2.0.0"
 
 dependencies:
+  firebase_core: ^0.8.0-nullsafety.0
+  firebase_core_platform_interface: ^3.0.2-nullsafety.0
+  firebase_crashlytics_platform_interface: ^1.2.0-nullsafety.0
   flutter:
     sdk: flutter
-  stack_trace: ^1.9.3
-  firebase_core: ^0.7.0
-  firebase_core_platform_interface: ^3.0.1
-  firebase_crashlytics_platform_interface: ^1.1.6
+  stack_trace: ^1.10.0-nullsafety.6
 
 dev_dependencies:
-  pedantic: ^1.8.0
   flutter_test:
     sdk: flutter
-  test: ^1.5.1
+  pedantic: 1.10.0-nullsafety.3
+  test: any
```

Comment:
Why the `any`?

Reply:
I thought that might be necessary after looking at the NNBD PRs for cloud functions and auth. I'll switch it back.